schema/journey.test.js

```diff
@@ -168,6 +190,112 @@ describe("schema", () => {
       });
   });
 
+  it("gets the shards of an existing class", () => {
+    return client.schema
+      .shardsGetter()
+      .withClassName(classObj.class)
+      .do()
+      .then((res) => {
+        res.forEach(shard => {
+          expect(shard.status).toEqual("READY");
+        });
+      });
+  })
+
+  it("updates a shard of an existing class to readonly", async () => {
+    var shards = await getShards(client, classObj.class);
```

Comment:
just to be 100% you would get the same shards information if you would instead of running `getShards` method you would run `client.schema.shardsGetter()...` yes?

Reply:
This `getShards` is just a thin wrapper around `client.schema.shardsGetter()...` so the test code is more readable 🙂 

See [here](https://github.com/semi-technologies/weaviate-javascript-client/blob/feature/WEAVIATE-4/schema/journey.test.js#L312)

Reply:
Yeah, now I see it clearly that you have wrapped this client call that you use for tests in more than one place